all-repos.yaml

```diff
@@ -227,3 +227,4 @@
 - https://github.com/charliermarsh/ruff-pre-commit
 - https://github.com/mrtazz/checkmake
 - https://github.com/jshwi/docsig
+- https://github.com/google/yamlfmt
```

Comment:
`yamlfmt .` is going to double-run every file (pre-commit works by passing the files as positional arguments to hooks).  there are two choices: (1) remove the `.` (recommended) or (2) set `pass_filenames: false` (I'm not accepting new things to the list which sets this)

Reply:
Makes sense, I'll remove that and re-open this PR on the next release.

Reply:
Looks like `v0.8.0` was tagged and removes the `.` [1].

[1] https://github.com/google/yamlfmt/blob/v0.8.0/.pre-commit-hooks.yaml

Reply:
nice!  can you move this up next to the other `google/` hook?

Reply:
Ah good idea, done in [afeac4c](https://github.com/pre-commit/pre-commit.com/pull/772/commits/afeac4c645a9276353d037f65e87807f01670c84)!